active_learning/tool_tracking_al/README.md

```diff
@@ -0,0 +1,50 @@
+# Active Learning Simulation Tutorial For Surgical Tool Segmentation
+
+Active learning (AL) is a machine learning framework that can serve as a data selection engine given an unlabeled pool of data. Active learning provides uncertainty estimation capabilities for trained deep learning models, in brief the models are able to assess about which predictions they are under or over confident. Based on the confidence the most relevant data for annotation can be selected. Thereafter annotation the labeled data is added to training pool of data for the model.
```

Comment:
Active learning provides uncertainty estimation capabilities for trained deep learning models, in brief the models are able to assess about which predictions they are under or over confident.

This seems two sentences to me.

Reply:
annotation the labeled data
->

annotation **of** the labeled data

Reply:
fixed.